include/musica/micm.hpp

```diff
@@ -140,11 +142,21 @@ namespace musica
     /// @param error Error struct to indicate success or failure
     void CreateRosenbrock(const std::string &config_path, Error *error);
 
+    /// @brief Create a BackwardEuler solver of vector-ordered matrix type by reading and parsing configuration file
```

Comment:
I have the same thoughts on the ordering of the functions related to the above comments.